### PR TITLE
[serious] Fixes buckling to operating tables

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Allows you to properly buckle folks to operating tables, notably locking them in if they're handcuffed as you'd expect with a chair.
(also updates some messages while I'm here).

## Why It's Good For The Game
This is behavior that you'd really expect to be the default, but currently isn't. At least in my own experience, I've run up against this a *ton*, with prisoners or research subjects crawling off even if they're handcuffed. This should make operating tables a bit more secure and workable.

## Testing
Buckled a monke to an optable, waited a bit.

## Changelog
:cl:
fix: You can now actually buckle people to operating tables.
/:cl:

### CODE REVIEW

File: code/game/machinery/OpTable.dm [hunk 1]
```diff
@@ -9,11 +9,13 @@
...
 	var/mob/living/carbon/patient
 	var/obj/machinery/computer/operating/computer
-	buckle_lying = -1
+	buckle_lying = TRUE
 	var/no_icon_updates = FALSE //set this to TRUE if you don't want the icons ever changing
 	var/list/injected_reagents = list()
 	var/reagent_target_amount = 1
 	var/inject_amount = 1
+	can_buckle = TRUE					// you can buckle someone if they have cuffs
+
```
Comment (Henri215) [inline code/game/machinery/OpTable.dm:18]:
```suggestion
	can_buckle = TRUE // you can buckle someone if they have cuffs
```

**Comment by Retrograde115** [comment]
How am I supposed to weasel my way out of my cuffs by lying to sec and telling them I have a storage implant now?

File: code/game/machinery/OpTable.dm [hunk 2]
```diff
@@ -9,11 +9,12 @@
 	active_power_consumption = 5
 	var/mob/living/carbon/patient
 	var/obj/machinery/computer/operating/computer
-	buckle_lying = -1
+	buckle_lying = TRUE
```
Comment (Sirryan2002) [inline code/game/machinery/OpTable.dm:12]: would you mind moving the inherited vars above the ones being defined on this type? same with `can_buckle`

Comment (lewcc) [reply code/game/machinery/OpTable.dm]: Addressed

**Review by Sirryan2002 — APPROVED** [review]

**Review by farie82 — APPROVED** [review]

**Pull request merged**